api-report/test-runtime-utils.api.md

```diff
@@ -69,7 +69,7 @@ export interface IMockContainerRuntimePendingMessage {
 export class InsecureTokenProvider implements ITokenProvider {
     constructor(tenantKey: string, user: IUser);
     // (undocumented)
-    fetchOrdererToken(tenantId: string, documentId: string): Promise<ITokenResponse>;
+    fetchOrdererToken(tenantId: string, documentId?: string): Promise<ITokenResponse>;
     // (undocumented)
     fetchStorageToken(tenantId: string, documentId: string): Promise<ITokenResponse>;
     }
```

Comment:
shouldn't `documentId` be an optional parameter in `fetchStorageToken`?

Reply:
I dont believe so. Its non-optional in `ITokenProvider` itself and I believe it is because if you are accessing storage, you are accessing an existing container from storage

Reply:
Don't you need a storage token in order to create a container?

Reply:
If this is true, that means we need to update `ITokenProvider`. Probably best to put that in a separate PR